src/manifest.json

```diff
@@ -17,6 +17,13 @@
       }
     },
 
+    "browser_action": {
+        "browser_style": true,
+        "default_icon": "fb-ban.png",
+        "default_title": "Block-facebook",
```

Comment:
Not sure where/if these titles are user visible, but should this be "Facebook Container"?